admin/installer/postgres/docker-compose-node1.yml

```diff
@@ -0,0 +1,12 @@
+postgres:
+  image: postgres
+  ports:
+     -  "5432:5432"
+  environment:
+     - "constraint:flocker-node==2"
+     - "POSTGRES_USER=flocker"
+     - "POSTGRES_PASSWORD=flocker"
```

Comment:
Thanks for raising the concern!
If we do not set POSTGRES_USER, it defaults to ``postgres`` as superuser account. This maps to a default ``postgres`` OS user, whose account is locked.

If we want to move away from hardcoding ``POSTGRES_USER`` and ``POSTGRES_PASSWORD``, here are the options:

1) Have tests use Docker instead of docker-compose, since Docker allows ``-e`` flag:

``docker run --name db -d -e POSTGRES_PASSWORD=flocker -e POSTGRES_USER=flocker postgres``

Docker-compose does not seem to have a way to pass in env variable from command line.

2) Move docker-compose yaml files into test branch and auto-generate them with secret password.

3) Explicitly call out changing ``POSTGRES_USER`` and ``POSTGRES_PASSWORD`` in installer documentation, as part of post-installation steps.

Will discuss the options with @wallrj and update the thread. Thanks!